modules/misc/news.nix

```
@@ -1039,6 +1039,13 @@ in
           A new module is available: 'programs.script-directory'.
         '';
       }
+
+      {
+        time = "2023-05-24T16:10:00+00:00";
+        message = ''
```

I imagine this is Linux only 🙂

```suggestion
        condition = hostPlatform.isLinux;
        message = ''
```